pkgs/applications/misc/obsidian/default.nix

```diff
@@ -19,6 +19,9 @@ appimageTools.wrapType2 rec {
     export XDG_DATA_DIRS=${gsettings-desktop-schemas}/share/gsettings-schemas/${gsettings-desktop-schemas.name}:${gtk3}/share/gsettings-schemas/${gtk3.name}:$XDG_DATA_DIRS
   '';
 
+  # Strip version from binary name.
+  extraInstallCommands = "mv $out/bin/${name} $out/bin/${pname}";
```

Comment:
I get mixed results with this... When running `nix-build -A obsidian` from the root of my cloned nixpkgs folder, the operation works correctly. However, in my /etc/nixos/config, if I import my custom branch and include obsidian, then it remains `obsidian-0.7.3`

Reply:
Found a good example in the notable package. Tested the fix on a seperate branch, got the expected name.